### PR TITLE
Fixes Issue #1697 by adding information to the special production con…

### PR DESCRIPTION
**Pull request by Kentalot**
…structions.

### CODE REVIEW

**Comment by Kentalot** [comment]
First shot at contributing.  Did some refactoring.  Hope it's ok!

**Comment by yairm210** [comment]
Looks ok!
Can you post a picture of it?

**Comment by Kentalot** [comment]
image

**Comment by Kentalot** [comment]
image

**Comment by Kentalot** [comment]
Oh, extra $ sign.  Pushed a fix.

File: core/src/com/unciv/logic/city/IConstruction.kt [hunk 1]
```diff
@@ -18,26 +20,29 @@ open class SpecialConstruction(override var name: String, val description: Strin
     override fun shouldBeDisplayed(construction: CityConstructions): Boolean {
         return isBuildable(construction)
     }
+    open fun getProductionTooltip(cityInfo: CityInfo) : String
+            = "\r\n${(cityInfo.cityStats.currentCityStats.production / CONVERSION_RATE).roundToInt()}/${"{turn}".tr()}"
```
Comment (yairm210) [inline core/src/com/unciv/logic/city/IConstruction.kt]: you can remove the {} brackets around "turn" since it's a single phrase

**Comment by yairm210** [comment]
This is good enough though =)

File: core/src/com/unciv/logic/city/IConstruction.kt (hunk 1 above)
Comment (Kentalot) [reply core/src/com/unciv/logic/city/IConstruction.kt]: Oh, I saw the brackets in the turnOrTurns method so assumed that was needed.  Thanks for the info.

**Pull request merged**